rust/parquet/src/column/writer.rs

```diff
@@ -892,8 +890,8 @@ mod tests {
         assert!(res.is_err());
         if let Err(err) = res {
             assert_eq!(
-                err.description(),
-                "Inconsistent length of definition and repetition levels: 3 != 2"
+                err.to_string(),
```

Comment:
In fact to_string is derived from Display trait. I would suggest to replace `to_string` with `display`. Though the results are same, but `display` matches with rust convention better.

Reply:
`err.display()` doesn't work, that's why I ended up resorting to `err.to_string()`. Would you be able to address the build failure instead? I can close this PR. 

As I mentioned in the PR, this is my first time coming across `quick-error` so I was bound to not do things correctly.

Reply:
Sorry, you should use `format!("{}", err)`. I took a look at the build failures mentioned in jira and I think your changes are good enough to resolve it.

Reply:
I've used `format!()` now